net.danielgm.CloudCompare.yaml

```diff
@@ -0,0 +1,290 @@
+app-id: net.danielgm.CloudCompare
+runtime: org.kde.Platform
+runtime-version: "5.12"
+sdk: org.kde.Sdk
+command: CloudCompare
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
Is there a reason of having host here? home should be enough I guess. Otherwise, just keep host instead of having both.

Reply:
I think `host` is required for accessing files from mounted devices or other partitions. Or is there a better way to express all mounted volumes (e.g. inside `/media`, `/mnt`, `/run` or `/afs`)?

Reply:
If you just need to open files and save them, portals should be the way to go. host gives you access to the whole host system. If you want only some specific folders/mounted devices, you should specify that.

Reply:
I had the issue that portals opened in the background when drag&drop a file into the app. By giving these access rights, the files are opened immediately.

Reply:
yeah, drag and drop is an issue: https://github.com/flatpak/xdg-desktop-portal/issues/99